### PR TITLE
Added elu activation function implementation

### LINKED ISSUE #19469: elu

### PR DESCRIPTION
**Pull request by MansiAmbalam**
Added implementation for elu based on the corresponding documentation.

Closes #19469

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -195,14 +195,9 @@ def relu_(x, name=None):
...
 @with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
 @to_ivy_arrays_and_back
-def elu(
-    x,
-    /,
-    *,
-    alpha=1.0,
-    name=None,
-):
-    return ivy.elu(x, alpha=alpha)
```
Comment (Daniel4078) [inline ivy/functional/frontends/paddle/nn/functional/activation.py:205]: Why did you remove the existing implementation? I think the ivy.elu (https://github.com/unifyai/ivy/blob/f5f8cd107865b7de6650e495474d0667e8ec4757/ivy/functional/ivy/experimental/activations.py#L408) is working alright?

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, you need to write a test function like the other pull requests linked in the TODO list this task is from. You can find other info about how to write our test in the [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues) and our [Deep Dive document](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html).

**Comment by Daniel4078** [comment]
By the way, you did not reference the issue you created to the TODO list issue we created to track the status of tasks.

**Comment by juliagsy** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**